mods/persistence/modules/client/preference_setup/preference_setup.dm

```diff
@@ -0,0 +1,285 @@
+var/global/const/CHARACTER_PREFERENCE_INPUT_TITLE = "Character Preference"
+
+/datum/category_group/player_setup_category/background_preferences
+	name = "Background"
+	sort_order = 1 // must go first because species
+	category_item_type = /datum/category_item/player_setup_item/background
+
+/datum/category_group/player_setup_category/physical_preferences
+	name = "Physical"
+	sort_order = 2
+	category_item_type = /datum/category_item/player_setup_item/physical
+
+/datum/category_group/player_setup_category/aspect_preferences
+	name = "Aspects"
+	sort_order = 3
+	category_item_type = /datum/category_item/player_setup_item/aspects
+
+/datum/category_group/player_setup_category/background_preferences/content(var/mob/user)
+	. = ""
+	for(var/datum/category_item/player_setup_item/PI in items)
+		. += "[PI.content(user)]<br>"
+/**
```

Comment:
First, again, you don't want to copy the whole thing just for changing one or two things.

And then, you can't do that. Its going to break preferences access down the line, I know I tried that before. If you want to remove elements from the menu, its going to need more than just commenting them like that.